docs/src/main/asciidoc/extension-authors-guide.adoc

```diff
@@ -494,15 +494,15 @@ public class FileConfig {
     String format;
 
     /**
-     * The file log level.
+     * The log level.
```

Comment:
Why remove the qualification here?  There are several different log level configuration items, so it's good to clarify what scope each applies to.

Reply:
@dmlloyd What you said is true in a broader view. There are more log level items in a configuration, but in this example there is only one, a `file`, so due to low level of details explained in this paragraph, having this qualifier was confusing in regards to the matter of this specific example (is this about a file `path`, or a `file` namespace). I can change this back, but only after adding more details to the description, i.e. "The level of logs to be written into the file". Please let me know if you accept this.

Reply:
Please see https://github.com/quarkusio/quarkus/pull/2589/commits/5ed96c05b1e317e973a3a3eeea0ed44a4a596ed3